### ISSUE TITLE
Support for external meshes

### ISSUE THREAD

**Issue by omichel** [cyberbotics/webots#1396]
In order to facilitate import from different 3D formats and to ease the maintenance of PROTO and world files, we should support external meshes with various format: DAE, STL, OBJ, FBX, etc.
The `Mesh` node would be a geometry node (like `IndexedFaceSet`) and could be defined as follow:

```
Mesh {
  MFString url [ ]       # URL of a 3D file in DAE, STL, OBJ or FBX format (similar to ImageTexture node)
  SFFloat creaseAngle 0  # used if no normals are provided (similar to IndexedFaceSet node)
  SFBool normalPerVertex TRUE # used if normals are provided (similar to IndexedFaceSet node)
}
```

Appearances and other 3D data found in DAE files should be ignored. Multiple meshes should be merged together.

In addition to the `Mesh` node, the `Shape` node should be extended to support a new MFString field called `url`, this field will allow specifying an external file (such as DAE, STL, OBJ, FBX) from which all the meshes and appearances are used.

**Comment by DavidMansolino** [comment]
Partially implemented in #1419.
A second PR should implement the `creaseAngle` and `normalPerVertex` fields (the Mesh node was already implemented in a way that it will be easy to add them).

**Comment by omichel** [comment]
I don't believe it's needed to add the `creaseAngle` and `normalPerVertex` to the `Mesh` node as most 3D files format (like OBJ) should include pre-computed normals. Using pre-computed normals makes that Webots loads meshes twice as fast as with a crease angle computation, which we want to encourage.
